setup.py

```diff
@@ -61,7 +60,7 @@ class sdist_with_git_version(sdist):
     def make_release_tree(self, base_dir, files):
         sdist.make_release_tree(self, base_dir, files)
         version_line = "__version__ = '{0}'\n".format(__version__)
-        orig_module = os.path.join(base_dir, 'sos/plugins/eucameta.py')
+        orig_module = "VERSION.py"
```

Comment:
Because this is missing `base_dir` it will edit the copy of VERSION.py in the source tree, not the build tree that will actually wind up in the tarball.  Run `setup.py sdist` from a clean source tree and compare the copies of that file in the source tree and the source tarball afterwards to reproduce this.